ci/debian/control

```diff
@@ -1,4 +1,4 @@
-Source: centreon-ha
+Source: centreon-ha-common
```

Comment:
You don't need to change the name of the project, it is even used to handle the source files. You can keep the name `centreon-ha`, what changes is below

Reply:
Oki doki!Very good.
I've put back the project names.
Is it good now?

Reply:
:smile: it is only here. the rest of the control files (install, dirs) you should keep with the package naming like you left before

Reply:
Oki doki. I had not understood well.
I'm sorry. 😅 
In Copyright, we keep the name centreon-ha when normally it's centreon-ha-common? 🤔

Reply:
> Oki doki. I had not understood well. I'm sorry. sweat_smile In Copyright file, we keep the name centreon-ha when normally it's centreon-ha-common? thinking

keep `centreon-ha` because is the name of project